### PR TITLE
[SOC] Dev run in Windows

### PR DESCRIPTION
**Pull request by QuakeWang**
Add some detail about building dev branch development environment in Windows environment

### CODE REVIEW

**Comment by QuakeWang** [comment]
@CalvinKirs PTAL

File: docs/zh-cn/dev/user_doc/dev_run.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+     4.   将 conf 目录下的 zoo_sample.cfg 文件，复制一份，重命名为 zoo.cfg，修改其中数据和日志的配置，如：
+
+          ```shell
+          dataDir=D:\\code\\apache-zookeeper-3.6.3-bin\\zkData ## 此处使用绝对路径
+          dataLogDir=D:\\code\\apache-zookeeper-3.6.3-bin\\zkLog
+          ```
+
+     5.   在 bin 中运行 zkServer.cmd，然后运行 zkCli.cmd 查看 zk 运行状态，可以查看 zk 节点信息即代表安装成功。
+
+3.   **搭建后端环境**
```
Comment (CalvinKirs) [inline docs/zh-cn/dev/user_doc/dev_run.md:34]: For the database, MySQL is not required, users can choose another(PGSQL). Suggest changes to related words

Comment (QuakeWang) [reply docs/zh-cn/dev/user_doc/dev_run.md:34]: > For the database, MySQL is not required, users can choose another(PGSQL). Suggest changes to related words

I have updated it, please rview~~

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**